package.json

```diff
@@ -25,6 +26,7 @@
     "solium": "^1.2.3",
     "truffle": "4.1.14",
     "truffle-bytecode-manager": "^1.1.1",
-    "truffle-extract": "^1.2.1"
+    "truffle-extract": "^1.2.1",
+    "web3-utils": "1.2.5"
```

Comment:
Let's create an issue to migrate to Truffle 5

Reply:
https://github.com/aragon/staking/issues/26